ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/test-your-equipment/test-your-equipment.component.ts

```diff
@@ -189,10 +192,30 @@ export class TestYourEquipmentComponent extends SuitabilityChoicePageBaseCompone
   protected bindModel(): void {
   }
 
-  replayVideo() {
-    if (this.pexipAPI) {
-      this.pexipAPI.disconnect();
-    }
+  async replayVideo() {
+    this.ngOnDestroy();
```

Comment:
Could we not call here ngOnDestroy, but just make its content as a method. Because it's a hook of Angular, confusing that we destroy component.